terraform/cloudrun.tf

```diff
@@ -57,6 +57,17 @@ resource "google_cloud_run_v2_service" "metrics" {
   }
 }
 
+resource "google_cloud_run_v2_service_iam_member" "public_metrics_access" {
+  project = var.project_id
+
+  location = google_cloud_run_v2_service.metrics.location
+  name     = google_cloud_run_v2_service.metrics.name
+  role     = "roles/run.invoker"
+  members = [
```

Comment:
Unlike `_iam_binding`, `_iam_member` does not allow a list of members as an argument. You will need to use a `for_each` instead, or "allUsers" directly.

Reply:
Was too fast at my documentation reading it seems, thanks